catboost/private/libs/algo_helpers/error_functions.h

```diff
@@ -10,6 +10,7 @@
 #include <catboost/private/libs/options/catboost_options.h>
 #include <catboost/private/libs/options/enums.h>
 #include <catboost/private/libs/options/restrictions.h>
+#include <catboost/private/libs/algo_helpers/survival_aft_utils.h>
```

Comment:
#include "survival_aft_utils.h" and higher

Reply:
fixed